examples/siro_sandbox/sandbox_app.py

```diff
@@ -457,6 +485,19 @@ def parse_debug_third_person(args, framebuffer_size):
         type=int,
         help="If specified, use the specified viewport height for the debug third-person camera",
     )
+    parser.add_argument(
+        "--enable-gfx-replay-save",
+        action="store_true",
+        default=False,
+        help="Save the gfx-replay keyframes to file. Use --gfx-replay-save-path to specify the save location.",
+    )
+    parser.add_argument(
+        "--gfx-replay-save-path",
```

Comment:
I don't feel strongly, but maybe it would be simpler to enable gfx-replay-save if and only if this save-path is specified? And thus we only need one new arg instead of two.

Reply:
I don't feel strongly either. Due to dependencies, I'll merge this into main as-is. We can change the command line arguments as needed.